root_VS2010/programs/C#/z_Common2.bat

```diff
@@ -3,13 +3,20 @@
 @rem --------------------------------------------------
 
 @rem --------------------------------------------------
+@rem Set Program Files path
+@rem --------------------------------------------------
+reg Query "HKLM\Hardware\Description\System\CentralProcessor\0" | find /i "x86" > NUL && set PROGRAM_FILES=Program Files || set PROGRAM_FILES=Program Files (x86)
+
+@rem --------------------------------------------------
 @rem Specifying Build tool.
 @rem --------------------------------------------------
-set BUILDFILEPATH="C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe"
+set BUILDFILEPATH="C:\%PROGRAM_FILES%\Microsoft Visual Studio 10.0\Common7\IDE\devenv.com"
 
-set BUILDFILEPATH2.0="C:\Windows\Microsoft.NET\Framework\v2.0.50727\MSBuild.exe"
-set BUILDFILEPATH3.5="C:\Windows\Microsoft.NET\Framework\v3.5\MSBuild.exe"
-set BUILDFILEPATH4.0="C:\Windows\Microsoft.NET\Framework\v4.0.30319\MSBuild.exe"
+set BUILDFILEPATH2.0="C:\%PROGRAM_FILES%\Microsoft Visual Studio 8\Common7\IDE\devenv.com"
```

Comment:
I wrote this to [here](https://github.com/OpenTouryoProject/OpenTouryo/issues/119#issuecomment-161265849).
Please merge.